tests/test_fixtures_valid.py

```diff
@@ -1,6 +1,10 @@
 # coding: utf-8
-from __future__ import (unicode_literals, print_function,
-                        absolute_import, division)
+from __future__ import (
+    unicode_literals,
+    print_function,
+    absolute_import,
+    division,
+)
```

Comment:
It seems I forgot to remove these imports. 
You can trash me since we don't support Python2 anymore